pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "prtpy",
     "pydot",
     "dataclasses-json",
-    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@main",
+    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@fix/issue_pce_200",
```

Comment:
Heads-UP: this will be reverted as soon as PR 141 lands (this was necessary to evaluate the tests here)

Reply:
Please be sure to push a tag to datamodel and use that.  Or use a short commit hash. Just do not refer to `datamodel@main`, because that is certain to break pce at some point.